zmon-common/src/main/java/org/zalando/zmon/domain/CheckDefinition.java

```diff
@@ -88,6 +89,9 @@
     @DatabaseField
     private DefinitionRuntime runtime;
 
+    @Transient
+    private String tier = "unknown";
```

Comment:
unknown should be a constant

Reply:
also, why not enum?

Reply:
also, what means "unknown"? Everything which is not "critical" or "important" is "other"

Reply:
We should explicitly set the tier. So the "unknown" means that we missed something(like we have no entity with tiers at all). Of course I could leave `null` by default, but for me "unknown" gives us a tip, but we will avoid NPE.

Reply:
Although, enum is a good choice, despite it requires a bit more code.

Reply:
Again, that's not correct. If we're missing an config entity, then all checks are "others". We should never ever have "unknown" entities

Reply:
perhaps something like "normal" or "low priority" would be better

Reply:
Once again, there're only 3 possible values of the tier: "critical", "important" and "others". The last one is default one. Any other value should not be there to avoid misunderstanding. Please refer to [the playbook ](https://docs.google.com/document/d/1RKMSY_QnRcvbHR4IrPjHyvn4D0E1mEJEARf1-ywJItY/edit?ts=5d761233#heading=h.4iw1wt93b12d)for more details.